docs/en/02_Developer_Guides/14_Files/05_File_Migration.md

```diff
@@ -90,7 +90,7 @@ It attempts to continue running to "complete" status even if it encounters error
 to ensure if everything went smoothly. Note that it's currently not possible to run specific subtasks via a queuedjob.
 
 While you can run the job directly through the CMS, it'll usually be more constrained by PHP `max_execution_time` settings.
-Many platforms such as the New Zealand Government Common Web Platform or SilverStripe Platform
+Many platforms such as the New Zealand Government Common Web Platform or Silverstripe CMS Platform
```

Comment:
Please change `Silverstripe CMS Platform` to `Silverstripe Cloud` :)

Reply:
Changed Silverstripe CMS Platform to Silverstripe Cloud 👍